docs/reference/commandline/service_logs.md

```diff
@@ -32,6 +32,76 @@ Options:
+redis.1.lt0yqr5wid01@centos7-x220-lan    |       _.-``    `.  `_.  ''-._           Redis 3.0.6 (00000000/0) 64 bit
+redis.1.lt0yqr5wid01@centos7-x220-lan    |   .-`` .-```.  ```\/    _.,_ ''-._                                   
+redis.1.lt0yqr5wid01@centos7-x220-lan    |  (    '      ,       .-`  | `,    )     Running in standalone mode
+redis.1.lt0yqr5wid01@centos7-x220-lan    |  |`-._`-...-` __...-.``-._|'` _.-'|     Port: 6379
+redis.1.lt0yqr5wid01@centos7-x220-lan    |  |    `-._   `._    /     _.-'    |     PID: 1
+redis.1.lt0yqr5wid01@centos7-x220-lan    |   `-._    `-._  `-./  _.-'    _.-'                                   
+redis.1.lt0yqr5wid01@centos7-x220-lan    |  |`-._`-._    `-.__.-'    _.-'_.-'|                                  
+redis.1.lt0yqr5wid01@centos7-x220-lan    |  |    `-._`-._        _.-'_.-'    |           http://redis.io        
+redis.1.lt0yqr5wid01@centos7-x220-lan    |   `-._    `-._`-.__.-'_.-'    _.-'                                   
+redis.1.lt0yqr5wid01@centos7-x220-lan    |  |`-._`-._    `-.__.-'    _.-'_.-'|                                  
+redis.1.lt0yqr5wid01@centos7-x220-lan    |  |    `-._`-._        _.-'_.-'    |                                  
+redis.1.lt0yqr5wid01@centos7-x220-lan    |   `-._    `-._`-.__.-'_.-'    _.-'                                   
+redis.1.lt0yqr5wid01@centos7-x220-lan    |       `-._    `-.__.-'    _.-'                                       
+redis.1.lt0yqr5wid01@centos7-x220-lan    |           `-._        _.-'                                           
+redis.1.lt0yqr5wid01@centos7-x220-lan    |               `-.__.-'                                               
+redis.1.lt0yqr5wid01@centos7-x220-lan    | 
+redis.1.lt0yqr5wid01@centos7-x220-lan    | 1:M 04 Jan 13:10:34.838 # WARNING: The TCP backlog setting of 511 cannot be enforced because /proc/sys/net/core/somaxconn is set to the lower value of 128.
+redis.1.lt0yqr5wid01@centos7-x220-lan    | 1:M 04 Jan 13:10:34.838 # Server started, Redis version 3.0.6
+redis.1.lt0yqr5wid01@centos7-x220-lan    | 1:M 04 Jan 13:10:34.838 # WARNING overcommit_memory is set to 0! Background save may fail under low memory condition. To fix this issue add 'vm.overcommit_memory = 1' to /etc/sysctl.conf and then reboot or run the command 'sysctl vm.overcommit_memory=1' for this to take effect.
+redis.1.lt0yqr5wid01@centos7-x220-lan    | 1:M 04 Jan 13:10:34.838 # WARNING you have Transparent Huge Pages (THP) support enabled in your kernel. This will create latency and memory usage issues with Redis. To fix this issue run the command 'echo never > /sys/kernel/mm/transparent_hugepage/enabled' as root, and add it to your /etc/rc.local in order to retain the setting after a reboot. Redis must be restarted after THP is disabled.
+redis.1.lt0yqr5wid01@centos7-x220-lan    | 1:M 04 Jan 13:10:34.838 * The server is now ready to accept connections on port 6379
+```
+### Log a task (--replica)
```

Comment:
I'd change this to

```markdown
### View logs for a task
```

(the `--replica` option is not related to this as well, so leave that out)